src/hw2/main.ts

```diff
@@ -0,0 +1,34 @@
+import { router } from './router';
+
+import express, { NextFunction, Request, Response } from 'express';
+import { ExpressJoiError } from 'express-joi-validation';
+
+const PORT = process.env.PORT || 3030;
+const app = express();
+
+app.use(express.json());
+
+app.use('/users', router);
+
+app.use(
+	(err: Error | ExpressJoiError, _: Request, res: Response, next: NextFunction) => {
```

Comment:
Usually all custom middlewares are stored in a separate folder <middleware>. And here you only apply it

Reply:
so should I move error handling middleware as exported function in another folder?